stock_inventory_virtual_location/models/stock_inventory_line.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.tools import float_utils
+
+
+class StockInventoryLine(models.Model):
+    _inherit = "stock.inventory.line"
+
+    virtual_location_id = fields.Many2one(
+        comodel_name='stock.location',
+        domain=[('usage', 'like', 'inventory')],
+        default=lambda self: self.product_id.property_stock_inventory,
+    )
+
+    @api.onchange('product_id', 'virtual_location_id')
+    def _onchange_product_id(self):
+        if self.product_id and not self.virtual_location_id:
+            self.virtual_location_id = \
+                self.product_id.property_stock_inventory
+
+    def _generate_moves(self):
+        moves = self.env['stock.move']
```

Comment:
Please add a hook here...

Reply:
I think its not needed. There isn't any other customization of this method in OCA.